packages/backend/ace-manifest.json

```
@@ -12,6 +12,17 @@
       "aliases": [],
       "flags": []
     },
+    "generate:commons": {
```

```suggestion
    "generate:commons": {
```
Это скорее contracts, а не commons. Конкретно здесь, а отдельный проект - common.